server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -483,23 +484,35 @@ private void copyDirectoryRecursively(File fromDir, File toDir) throws IOExcepti
         }
     }
 
-    private void stripBitcode(File file) throws ExtenderException {
-        String command = String.format("bitcode_strip %s -r -o %s", file.getAbsolutePath(), file.getAbsolutePath());
+    private void stripBitcode(File file, PlatformConfig config) throws ExtenderException {
+        String command = null;
+
+        if (config.bitcodeStripCmd != null) {
+            Map<String, Object> context = new HashMap<>(config.context);
+            context.put("source", file.getAbsolutePath());
+            context.put("target", file.getAbsolutePath());
+
+            command = templateExecutor.execute(config.bitcodeStripCmd, context);
+        }
+        else {
+            command = String.format("bitcode_strip %s -r -o %s", file.getAbsolutePath(), file.getAbsolutePath());
+        }
```

Comment:
Please write down the defold version in which this became the fallback, so that we can more easily remove it later on.